fbw-common/src/systems/instruments/src/ND/TcasWxrMessages.tsx

```diff
@@ -66,19 +72,32 @@ export class TcasWxrMessages extends DisplayComponent<TcasWXMessagesProps> {
       .on('tcasFault')
       .whenChanged()
       .handle((value) => this.failSub.set(value));
+    sub
+      .on('tcasMode')
+      .whenChanged()
+      .handle((value) => this.tcasModeSub.set(value));
 
     MappedSubject.create(
-      ([taOnly, failed]) => {
+      ([taOnly, failed, mode]) => {
         if (failed) {
-          this.leftMessage.set({ text: 'TCAS', color: 'Amber' });
+          this.leftMessage.set({ text: 'TCAS' });
+          this.textClass.set('Amber TCASMessage');
+          this.rectClass.set('Grey BackgroundFill');
         } else if (taOnly) {
-          this.leftMessage.set({ text: 'TA ONLY', color: 'White' });
+          this.leftMessage.set({ text: 'TA ONLY' });
+          this.textClass.set('White TCASMessage');
+          this.rectClass.set('Grey BackgroundFill');
+        } else if (mode === 0) {
+          this.leftMessage.set({ text: 'TCAS STBY' });
+          this.textClass.set('White TCASMessage TCASStby');
+          this.rectClass.set('Grey BackgroundFill TCASStby');
```

Comment:
Classes should be either a SetSubject, or an object in inline in the JSX (preferred declarative approach).